Cargo.toml

```diff
@@ -18,6 +18,8 @@ serde = {version = "1.0.102", features=["std", "derive"], optional = true}
 pyo3 = {version = "0.16", features=["extension-module", "abi3-py37"], optional = true }
 wasm-bindgen = {version = "0.2", optional = true}
 js-sys = {version = "0.3.60", optional = true}
+anyhow = {version = "1.0.65", optional = true}
+nom = {version = "7.1.1", optional = true}
```

Comment:
I'd rather not have these dependencies. Are they really necessary?

Reply:
Thanks for the comment! `anyhow` was removed. `nom` is needed.

Reply:
`RaptorqFrame` allows to get frame details when you're wroking with the api. It allows to develop progress bars and show the entire progress of the scanning.

Reply:
I don't see why it needs to be in this crate though. Users can already get the size of the object they're transmitting (since they must have a reference to it when encoding), and they also can get the total number of packets since the encoder outputs them all in a Vec.
I think it's best for users to transmit those out-of-band for use in building a progress bar

Reply:
I haven't found the way it in the client application like Javascript web app. However if you insist on client bits frame processing then NOM and RaptorqFrame may be removed.

Reply:
I mean that the sender should provide a separate API which returns the length of the object and the number of packets. Then the client can just count the number of raptorq packets that it has received and use that as a progress bar

Reply:
I got it! I made changed 👍